themes/bootstrap5/templates/search/advanced/build_page.phtml

```diff
@@ -10,6 +10,7 @@ $(document).ready(function() {
   $('#new_group_template').addClass('hidden').detach().appendTo('[role="main"]');
   $('#new_group_template').find('.adv-group').removeAttr('id');
   $('#new_group_template').find('.form-control').removeAttr('id');
+  $('#new_group_template').find('.form-select').removeAttr('id');
```

Comment:
This was missed during the change from `form-control` to `form-select` class for select fields.